src/nl/coolcat007/netbeansmodules/goprojecttype/GoProject.java

```diff
@@ -207,7 +207,9 @@ public String getDisplayName() {
         @Override
         public Node findPath(Node root, Object target) {
             //leave unimplemented for now
-            return null;

```

Comment:
"Node" is an abstract class used by NetBeans. Need to find the right way to implement it.